### ISSUE TITLE
Typescript install seems not available in the wiki

### ISSUE THREAD

**Issue by Misaka-0x447f** [goriunov/ClusterWS-Client-JS#56]
### Submitting

- [ ] Bug
- [x] Question
- [ ] Suggetion
- [ ] Other

### Details
Clicked typescript link and nothing changed.

**Comment by goriunov** [comment]
@Misaka-0x447f  yes documentation for typescript has not been completed for front end library (it is not much different from js and you can always check index.d.ts file for typings)

**Comment by Misaka-0x447f** [comment]
So can just write 'no additional type definition package install required for typescript' instead of placing a typescript link? I found that it comes with types after create this issue.

**Comment by goriunov** [comment]
Yes that that is a good idea, will try to do it soon :)
